tests/common/plugins/test_completeness/__init__.py

```diff
@@ -4,10 +4,45 @@
 import warnings
 
 class CompletenessLevel(enum.IntEnum):
-    debug = 0 # Minimum execution
-    basic = 1
-    confident = 2
-    thorough = 3 # Maximum execution
+    diagnose = 0
```

Comment:
you did mention this is an unordered level. do you need to shift the other levels in that case? Wouldn't specifying some value for 'diagnose' level suffice?

Reply:
Any `int` value can be assigned to `diagnose`. It has to be `int` due to base type `enum.IntEnum`.
During normalization, there is a special handling for `diagnose` which does not consider ordering.

Reply:
I started with `enum.IntEnum` as the levels were assumed to follow a ordered priority. But this does not seem to be the case anymore. Removing base class inheritance will make this list look simple, but then we downside is we will have to implement methods to handle ordered levels, exists` check, etc. These methods are presently used from `enum.IntEnum`

Reply:
Just to keep the change minimal, you can assign diagnose some int value since its treated differently

Reply:
Do you mean any `int` value other than `0`?

Reply:
yes. some large value leaving enough space after max level in case we need to add more ordered levels in future